RELEASE-NOTES.rst

```diff
@@ -19,12 +19,12 @@ What's new
 Installation
 ------------
 
-   $ pip install invenio-sequencegenerator==0.1.0
+   $ pip install invenio-sequencegenerator==1.0.0a1
```

Comment:
There are more versions and dates that should be updated (see the top of the file).